wrapper/src/main/java/software/amazon/jdbc/util/AsynchronousMethodsHelper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.util;
+
+import java.util.Collections;
+import java.util.List;
+
+public class AsynchronousMethodsHelper {
+  public static final List<String> ASYNCHRONOUS_METHODS = Collections.singletonList(
+      "Statement.cancel"
```

Comment:
What about other variations of statement.cancel? e.g. PreparedStatement.cancel

Reply:
Good point, it's been added to the list